src/Util.ts

```diff
@@ -250,15 +204,18 @@ export function getLocalREADMEFile() {
 }
 
 export function setAuthCallbackState(value: string) {
-  fileIt.setJsonValue(getDeviceFile(), "auth_callback_state", value);
+  setJsonItem(getDeviceFile(), "auth_callback_state", value);
 }
 
 export function getLogId() {
   return 'MusicTime';
 }
 
-export function logIt(message: string) {
+export function logIt(message: string, isError: boolean = false) {
   outputChannel.appendLine(`${formatISO(new Date())} ${getLogId()}: ${message}`);
+	if (isError) {
```

Comment:
weird indentation here or is this GitHub showing it weird?

Reply:
yeah, i think its github but I've updated the package with eslint and cleaned up the files